### PR TITLE
runelite-api & runescape-api: Rename Area to MapIcon

### PR DESCRIPTION
**Pull request by MESLewis**
Rename API methods in line with internal class renaming

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/Client.java [hunk 1]
```diff
@@ -223,15 +223,15 @@
 
 	NPCComposition getNpcDefinition(int npcId);
 
-	Area[] getMapAreas();
+	MapIcon[] getMapIconTypes();
```
Comment (LeviSchuck) [inline runelite-api/src/main/java/net/runelite/api/Client.java]: I'm a little worried about changing anything in the `Client.java` as this has to mixin with the runescape client.

Comment (MESLewis) [reply runelite-api/src/main/java/net/runelite/api/Client.java]: I've done testing with this on an internal branch, the renaming is necessary to accurately reflect what is being returned.

Comment (MESLewis) [reply runelite-api/src/main/java/net/runelite/api/Client.java]: Furthermore all the renamed methods are unused in the current code base

**Comment by MESLewis** [comment]
Any update on this? I will resolve conflicts if its being considered

**Comment by devLotto** [comment]
Area was renamed to MapElementConfig in 11ab3dc2a2f10d10abcaf5ed0e861ae6c59a0229

**Pull request closed without merging**